Storage.js

```diff
@@ -0,0 +1,20 @@
+const inputemail = document.getElementById('email');
+const inputname = document.getElementById('name');
+const formst = document.getElementById('form');
+const arr = [];
+const Localstore = (e) => {
+  e.preventDefault();
+  const name = inputname.value;
+  const email = inputemail.value;
+
+  const obj = {};
+
+  obj.Name = name;
+  obj.Email = email;
+
+  const result = JSON.stringify(obj);
+  arr.push(result);
+  localStorage.setItem('data', JSON.stringify(arr));
+};
+
+formst.addEventListener('submit', Localstore);
```

Comment:
- [ ]  Please take into account that you need to save the data before the form is submitted and you also need to retrieve your data. I made a video explaining  what i mean, please check it and feel free to ask me any question you have.

### This is the link to the video 
https://www.loom.com/share/6430f6e092714bb4931efabe9f3f863d